core/src/autogluon/core/scheduler/seq_scheduler.py

```diff
@@ -0,0 +1,147 @@
+import logging
+import time
+from collections import OrderedDict
+from copy import deepcopy
+
+from autogluon.core.searcher import GPFIFOSearcher
+from autogluon.core.utils import EasyDict
+from tqdm.auto import tqdm
+
+logger = logging.getLogger(__name__)
+
+
+class LocalReporter:
+    def __init__(self, trial, config, training_history: dict, config_history: dict):
+        self.trial = trial
+
+        self.training_history = training_history
+        if trial not in self.training_history:
```

Comment:
I think it would be a mistake here if trial was already in training_history ?

Reply:
Removed